pom.xml

```diff
@@ -3,7 +3,7 @@
 
   <groupId>com.joyent.manta</groupId>
   <artifactId>java-manta</artifactId>
-  <version>1.5.4</version>
+  <version>1.6.0</version>
```

Comment:
We should be using SNAPSHOT versions and then change to a non-snapshot version during the release process.
ie... 1.6.0 should be 1.6.0-SNAPSHOT until release time.